src/asmcnc/apps/wifi_app/wifi_documentation/Francais (FR).rst

```diff
@@ -1,74 +1,74 @@
-
-* Certains routeurs offrent un choix entre 2,4 GHz et 5 GHz canaux. 5 canaux GHz peuvent être moins fiables, donc nous vous conseillons de connecter sur le canal 2,4 GHz lorsque cela est possible.
-
-
-Connexion via un point d'accès mobile
--------------------------------------
-
-Nous avons vu des problèmes en essayant de se connecter sur les appareils Apple. Nous vous conseillons connecter sur Android lorsque cela est possible.
-
-
-Appareils Apple
-~~~~~~~~~~~~~~~~
-
-Pour vous connecter à un appareil Apple, essayez les étapes suivantes:
-
-* Mise hors tension SmartBench
-
-* Redémarrer iPhone
-
-* Au redémarrage de l'iPhone, le câble de connexion USB de l'iPhone dans le port USB de la console SmartBench
-
-* Allumez votre iPhone hotspot (Paramètres > hotspot personnel)
-
-* Puissance sur SmartBench
-
-* Entrez vos informations à l'écran des paramètres du réseau
-
-.. _bottom:
+.. _top:
+
+Conseils de connexion
+=====================
+
+Si vous rencontrez des problèmes pour vous connecter la console à votre réseau wifi, lire ces notes:
+
+
+Le nom du réseau et le mot de passe
+-----------------------------------
+
+* **Le  mot de passe doit être compris entre 8 et 63 caractères.**
+
+* Les deux sont sensibles à la casse, assurez-vous qu'ils sont exacts.
+
+* Évitez les caractères spéciaux.
+
+* Assurez-vous qu'il n'y a pas d'espaces supplémentaires.
+
+
+Code postal
+-----------
+
+Assurez-vous que la correspondance du code pays. Par défaut, c’est GB qui est assez universel, mais qui peut ne pas convenir à certaines régions. Voir la section Codes pays ci-dessous pour celui qui correspond à votre région.
+
+
+Avant d’appuyer sur « Connecter »
+---------------------------------
+
+* Vérifiez tous les détails sont exacts.
+
+* En relâchant le bouton Connecter, la console devra redémarrer avant de tenter d'établir une connexion en utilisant le nom du réseau et les détails de mot de passe saisi.
+
+
+Il est temps de se connecter
+----------------------------
+
+Certains routeurs peuvent prendre plusieurs minutes pour se connecter.
+
+
+Connexion via un routeur wifi
+-----------------------------
+
+* Si vous rencontrez des problèmes, réinitialiser et redémarrer le routeur.
+
+* Assurez-vous que la console est à portée - vous pouvez utiliser une application du scanner wifi pour vérifier le signal.
+
+* Certains routeurs offrent un choix entre des canaux 2,4 GHz et 5 GHz. Le canal 5 GHz peut être moins fiables, donc nous vous conseillons de vous connecter sur le canal 2,4 GHz lorsque cela est possible.
+
+
+Connexion via un point d'accès mobile
+-------------------------------------
+
+Nous avons noté des problèmes de connexions sur les appareils Apple. Nous vous conseillons de vous connecter sur Android lorsque cela est possible.
+
+
+Appareils Apple
+~~~~~~~~~~~~~~~
+
+Pour vous connecter à un appareil Apple, essayez les étapes suivantes :
```

Comment:
think there's an extra space here

Reply:
Not sure what you mean here, could you clarify?

Reply:
between suivantes and the ":", seems there's an extra space